autoprotocol/protocol.py

```diff
@@ -1677,7 +1679,7 @@ def acoustic_transfer(
             if v > Unit(0, "microliter"):
                 transfers.append(xfer)
             if self.propagate_properties:
-                d.set_properties(s.properties)
+                d.add_properties(s.properties, silence_warnings=True)
```

Comment:
The warning dependency is on line 12 in this module. Instead of changing the signature of the add_properties method, what about using `warnings.filterwarnings("ignore")` instead. If this is specific to acoustic_transfer and will be true every time then we should be able to do something like:
```python
if self.propagate_properties:
     warnings.filterwarnings("ignore")
     d.add_properties(s.properties)
```
Or you could put it in the application code to filter the warnings that are coming from your acoustic_transfer call
```python

def my_package_protocol(protocol, params):
    ...
    warnings.filterwarnings("ignore")
    protocol.acoustic_transfer(...)

Reply:
That was my initial plan, but josh nor I could figure out how to test if it was _actually_ silencing the warnings. We spent too long on it, and this was just as effective. And wouldn't `filterwarnings` filter other possible warnings? This method just skips the one `warn` call

Reply:
Use the https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings the warnings module has to catch the warnings from your package protocol code.
```python
def my_package_protocol(protocol, params):
    ...
    with warnings.catch_warnings():
         protocol.acoustic_transfer(...)
```